### ISSUE TITLE
Can ML-images datasets perform multiple output detection?

### ISSUE THREAD

**Issue by Easyfeng222** [Tencent/tencent-ml-images#47]
Thanks for your contribution, I read your paper and said that the verification result is in a single-label dataset. Can the ML-images dataset be multi-output tested? Is it convenient to open source if there is one?

**Comment by wubaoyuan** [comment]
@Easyfeng222 Constructing a clean-annotated, large-size, same-distribution-with-training multi-labeled validation set is very very costly. We are trying to enlarge the current validation set. Please refer to Sc 4.2 for a brief illustration. We believe that the transfer learning to single-label dataset can also validate the quality of the ML-Images.

**Comment by Easyfeng222** [comment]
@wubaoyuan Thank you for your quick reply.the first ckpt is pre-trained with the ML-Images dictionary. Can you add a demo to use this ckpt as Single label classification.I am very interested in the demo effect of ML-images.Looking forward to your reply
